guide.md

```diff
@@ -168,7 +168,44 @@
 * built-in security (authentication and authorization)
 * multiplexing (through the use of channels)
 * HTML5 real-time enabled templating engine (xRTML)
-	
+
+#### [Streamdata.io](http://streamdata.io)
+
+* [Docs](http://streamdata.io/doc/)
+* [Pricing](http://streamdata.io/pricing/)
+* [Libraries](https://github.com/streamdataio/)
+
+> Streamdata.io is a real-time cache proxy allowing you to poll JSON REST APIs and push updates to clients. But wait, there is more: Streamdata.io keeps a history of modifications that occur on the data between two pollings! This way, streamdata.io is able to give you the list of modifications which happened since last time you fetched the data.
+
+* Server-Sent Events
+* Fallback-support
+* REST
+* PubSub
+* Proxy
+* JavaScript
+* iOS
+* Java
+* Android
+
+##### JavaScript (client)
```

Comment:
Thanks for the PR!

I'm planning on removing the code samples as it's a pain to maintain. Can you please update the PR to remove these? Then I'll merge and push to my site :smile_cat: